### PR TITLE
feat: add support for deb templates

### PR DESCRIPTION
**Pull request by vandot**
Add support for `templates` file inside `DEBIAN` that is needed for `debconf` to be able to ask question user during installation. More on subject [here](http://www.fifi.org/doc/debconf-doc/tutorial.html) and [here](https://manpages.debian.org/testing/debconf-doc/debconf-devel.7.en.html)

### CODE REVIEW

**Comment by djgilcrease** [comment]
Is there a reason this cannot just be handled with the normal files process?

**Comment by vandot** [comment]
@djgilcrease `templates` file needs to be inside `DEBIAN` folder, maybe I missed something but with `files` and `conffiles` files are placed in relative installation path

File: nfpm.go [hunk 1]
```diff
@@ -204,6 +204,7 @@ type APKSignature struct {
 // Deb is custom configs that are only available on deb packages.
 type Deb struct {
 	Scripts         DebScripts   `yaml:"scripts,omitempty"`
+	Templates       string       `yaml:"templates,omitempty"`
```
Comment (djgilcrease) [inline nfpm.go:207]: I would move this into the `DebScripts` object

**Review by djgilcrease — APPROVED** [review]
LGTM

**Review by caarlos0 — APPROVED** [review]
can you add it to the docs too?

other than that LGTM

**Comment by vandot** [comment]
Done, added to the docs.

**Pull request merged**